Automattic/wp-calypso
# [e2e] Highlight elements during test execution

#### Changes proposed in this Pull Request

Highlight elements ( for clicks and text typing ) during test execution. This is something which we had before [#](https://github.com/Automattic/wp-e2e-tests/pull/190) [#](https://github.com/Automattic/wp-e2e-tests/pull/182), but it was [removed](https://github.com/Automattic/wp-e2e-tests/pull/1246) and it was turned off by default. Now it's turned on by default for all test runs. 

For now, [IE canaries look good](https://450637-45936895-gh.circle-artifacts.com/0/tmp/artifacts/screenshots/screenshots/FAILED-EN-DESKTOP-can-see-the--site-title--page--and-enter-the-site-title-1570541202329.png). If we see any problem there we can easily turn this off for IE.

For canaries highlighting elements is enabled through setting up env var in [CircleCI config](https://github.com/Automattic/wp-calypso/blob/160d9423cc24e93b630dfc65b76376791049cc13/.circleci/config.yml#L156). 
For full suite against live branches, env var is set [here](https://circleci.com/gh/Automattic/wp-e2e-tests-for-branches/edit#env-vars).

#### Testing instructions

* Make sure that all tests are passing in CircleCI, full suite and canaries.
* If some test fail, take a look at the video of the test execution. Are the elements highlighted?
* Run any test locally. Are the elements highlighted?
